cnf-app-mac-operator/Makefile

```diff
@@ -125,7 +131,7 @@ help: ## Display this help.
 # In manifests, removing the automatic creation of rbac:roleName=manager-role, this will be created manually
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
As said in the comments above, we don't use `rbac:roleName=manager-role`, this needs to be removed